Odata-docs/connectedservice/getting-started.md

```diff
@@ -41,6 +47,9 @@ After successful completion, you should see a **Connected Services** section und
 
 ![TripPin Connected Service added to project](../assets/2020-03-06-OCS-added-to-project.png)
 
+> [!NOTE]
+> OData Connected Service installs the latest versions of OData core libraries (i.e. OData.NET) if they are not installed. OData.NET 8 libraries support .NET 8 and later, they do not support older versions of .NET Core or any version of .NET Framework. If your project is targetting any of these .NET versions, then you should manually install Microsoft.Spatial 7.x Microsoft.OData.Edm 7.x, Microsoft.OData.Core 7.x and Microsoft.Client 7.x.
```

Comment:
I don't think we need to list all 4 dependent libraries. We should instruct the customer to install Microsoft.OData.Client 7.x. When that library is installed, it will install the dependent libraries